portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+  const QUOTES =
+      ['Toit Nups!',
+       'NINE-NINE!',
+        'Cool cool cool cool cool, no doubt no doubt.',
+         'Why is no one having a good time? I specifically requested it.'];
+
 /**
  * Adds a random quote from Brooklyn-99 to the page.
  */
 function addRandomQuote() {
-  const quotes =
-      ['Toit Nups!', 'NINE-NINE!', 'Cool cool cool cool cool, no doubt no doubt.', 'Why is no one having a good time? I specifically requested it.'];
-
   // Pick a random greeting.
-  const quote = quotes[Math.floor(Math.random() * quotes.length)];
+  const quote = QUOTES[Math.floor(Math.random() * QUOTES.length)];
 
   // Add it to the page.
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+
+
+/**
+ * This function opens the tab we want to access and hides the unnecessary tabs
+ */
+function openTab(evt, tabName) {
+    var tabcontent = document.getElementsByClassName("tabcontent");
```

Comment:
use const for each of these variables.
Anything that can be made const without erroring should be.

These don't need to be top-level consts (since the result of getElementsByClassName("tablinks") might theoretically change), but since the value won't change during this function at all, you can use const